### PR TITLE
chore(deps): update docusaur

### PR DESCRIPTION
**Pull request by flavio**
Use latest version of docusaur

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2349,7 +2349,7 @@
     "@docsearch/css" "3.1.1"
     algoliasearch "^4.0.0"
 
-"@docusaurus/core@2.2.0", "@docusaurus/core@^2.2.0":
+"@docusaurus/core@2.2.0":
```
Comment (viccuad) [inline yarn.lock:2352]: I would have expected 2.2.0 and its dependencies to get dropped. Is that ok?

**Review by jvanz — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (jvanz) [reply yarn.lock:2352]: Good question. I think yarn should does that. @jordojordo do you know if this is okay?

Comment (btat) [reply yarn.lock:2352]: It's possible `"@docusaurus/theme-search-algolia": "^2.2.0"` not being bumped is causing this.

File: package.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.2.0",
-    "@docusaurus/preset-classic": "^2.2.0",
+    "@docusaurus/core": "^2.3.1",
+    "@docusaurus/preset-classic": "^2.3.1",
     "@docusaurus/theme-search-algolia": "^2.2.0",
```
Comment (btat) [inline package.json:19]: I think this needs to be bumped as well. The [docs](https://docusaurus.io/docs/installation#updating-your-docusaurus-version) mention 

> Note that all `@docusaurus/`-namespaced packages should be using the same version.

File: yarn.lock (hunk 1 above)
Comment (jordojordo) [reply yarn.lock:2352]: @btat is right, running `yarn list @docusaurus/core` yields:
```console
$ yarn list @docusaurus/core
yarn list v1.22.19
├─ @docusaurus/core@2.3.1
└─ @docusaurus/theme-search-algolia@2.2.0
   └─ @docusaurus/core@2.2.0
```

File: package.json (hunk 2 above)
Comment (flavio) [reply package.json]: Good catch, this should be fixed now

File: yarn.lock (hunk 1 above)
Comment (flavio) [reply yarn.lock]: I fixed that

**Comment by flavio** [comment]
Thanks for the reviews, please take another look to ensure everything is fine.

**Review by viccuad — APPROVED** [review]
LGTM!

**Pull request merged**